DDL.sql

```diff
@@ -0,0 +1,109 @@
+-- 20 April 2020
+
+CREATE SCHEMA IF NOT EXISTS ${schema};
+
+-- ---------------------------------------------------------------------------
+-- Table `ecm`
+--
+-- For the purpose of the tests, the following fields should be populated
+-- for use by ecm_prep.py and run.py: id, name, definition
+-- ---------------------------------------------------------------------------
+CREATE TABLE IF NOT EXISTS ${schema}.ecm
+(
+    id             integer PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name           varchar(255) UNIQUE NOT NULL,
+    definition     jsonb               NOT NULL,
```

Comment:
```suggestion
    definition      jsonb               NOT NULL,
    load_shape_file varchar(150),
    climate_zone    varchar(150),
    building_class  varchar(150),
    end_use         varchar(150),
```
Note that you might want to reindent the other row definitions to line up with these lines.